### PR TITLE
Fix capitalization of GitHub org

### PR DESCRIPTION
**Pull request by ChaseKnowlden**
`github.com/Microsoft` -> `github.com/microsoft`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Visual Studio Code - Open Source ("Code - OSS")
 [![Build Status](https://dev.azure.com/vscode/VSCode/_apis/build/status/VS%20Code?branchName=master)](https://aka.ms/vscode-builds)
-[![Feature Requests](https://img.shields.io/github/issues/Microsoft/vscode/feature-request.svg)](https://github.com/Microsoft/vscode/issues?q=is%3Aopen+is%3Aissue+label%3Afeature-request+sort%3Areactions-%2B1-desc)
-[![Bugs](https://img.shields.io/github/issues/Microsoft/vscode/bug.svg)](https://github.com/Microsoft/vscode/issues?utf8=✓&q=is%3Aissue+is%3Aopen+label%3Abug)
+[![Feature Requests](https://img.shields.io/github/issues/vscode/feature-request.svg)](https://github.com/microsoft/vscode/issues?q=is%3Aopen+is%3Aissue+label%3Afeature-request+sort%3Areactions-%2B1-desc)
```
Comment (ItzLevvie) [inline README.md:3]:
```suggestion
[![Feature Requests](https://img.shields.io/github/issues/microsoft/vscode/feature-request.svg)](https://github.com/microsoft/vscode/issues?q=is%3Aopen+is%3Aissue+label%3Afeature-request+sort%3Areactions-%2B1-desc)
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ItzLevvie) [reply README.md:3]: Removing this breaks Shield from finding this repository.

**Review by ItzLevvie — APPROVED** [review]
LGTM.

File: README.md (hunk 1 above)
Comment (ChaseKnowlden) [reply README.md]: i fixed a shield

**Review by aeschli — CHANGES_REQUESTED** [review]

**Comment by aeschli** [comment]
I fixed the merge conflicts but didn't had the permission to write back to the fork.
So I directly pushed to master: https://github.com/microsoft/vscode/commit/e8760a5d6c0e09ab692e5b3e414eca018745490f and more...

Thanks @ChaseKnowlden !

**Pull request closed without merging**